website/docs/api/themes/theme-configuration.md

```diff
@@ -208,6 +208,25 @@ You can add items to the navbar via `themeConfig.navbar.items`.
 
 By default, Navbar items are regular links (internal or external).
 
+React Router should automatically apply active link styling to links, but you can use `activeBasePath` in edge cases. For cases in which a link should be active on several different paths (such as when you have multiple doc folders under the same sidebar), you can use `activeBaseRegex`. `activeBaseRegex` is a more flexible alternative to `activeBasePath` and takes precedence over it -- Docusaurus parses it into a regular expression that is tested against the current URL.
```

Comment:
What about creating a "navbar link" section?

you'd be able to link to it when mentioning "link-like"

Reply:
the "Navbar items" section could be a more general presentation of the navbar items system

Reply:
Bonne idée👍